.dockerignore

```diff
@@ -1,7 +1,6 @@
 .env
-**/*.sqlite3
+*.log
+*.sqlite3
+__pycache__/
 
-**/__pycache__
-**/*.pyc
-
-**/*.log
+/.install.*.bash.inc
```

Comment:
These changes are mostly, I think, just simplification – but also, the development bootstrap script, `develop`, caches its dependency at that file, `.install.*.bash.inc`.